scripts/release/utils.js

```diff
@@ -103,6 +104,22 @@ const getChecksumForCurrentRevision = async cwd => {
   return hashedPackages.hash.slice(0, 7);
 };
 
+const getCurrentDateString = () => {
+  const date = new Date();
+  const year = date.getFullYear();
+  let month = date.getMonth() + 1;
+  let day = date.getDate();
+
+  if (month < 10) {
+    month = `0${month}`;
+  }
+  if (day < 10) {
+    day = `0${day}`;
+  }
+
+  return `${year}${month}${day}`;
```

Comment:
Any reason not using toString with a format specifier? Either works, I’m only curious if it doesn’t work for this use case.

Reply:
I'm unfamiliar with the approach you're referring to. DateFormatter seemed like overkill if that's what you mean. But show a snippet?